src/frontend/views/invoice.njk

```diff
@@ -30,6 +33,21 @@
     <strong>" + contractName + "</strong>",
     classes: bannerClass
     }) }}
+  <div class="govuk-inset-text">
+    {{ govukAccordion({
+        id: "accordion-default",
+        items: [
+        {
+        heading: {
+        text: "Status guidance"
+        },
+        content: {
+        html: statusGuidance
+        }
+        }
+        ]
+        }) }}
+  </div>
```

Comment:
is this really what the linter did to this?

Reply:
yup!